### PR TITLE
Fix #1283: Make install, build, and serve one command each.

### LINKED ISSUE #1283: Make install, build, and serve one command each.
### Install

```
sudo apt install php7.0-fpm php7.0-cli php7.0-curl php7.0-intl php7.0-json php7.0-sqlite php7.0-mbstring php7.0-xml &&
sudo apt install nodejs &&
npm install
```

  - `install:composer` => `cd backend && composer install`
  - `install:submodule` => `git submodule init && git submodule update --force --rebase`
  - `postinstall` => `npm run install:submodule && npm run install:composer`

### Build
`npm run build`

### Serve
`npm run start`

### PR DESCRIPTION
**Pull request by lewisgoddard**
Fixes #1283

### Changes Summary

- [x] Install with `npm install`
- [x] Build with `npm run build`
- [x] Server with `npm run start`
- [x] Document

This pull request is ready for review.

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -1,35 +1,50 @@
...
-* `php7.0-json`
-* `php7.0-mbstring`
-* `php7.0-xml`
-* packages installed with `npm install`
+Alternatively, for a simple local development environment running on PHP, you will need:
+
+* [A supported version of PHP](http://php.net/supported-versions.php) with:
+  * `php7.0-cli`
+  * `php7.0-fpm`
+  * `php7.0-curl`
```
Comment (btkostner) [inline .github/CONTRIBUTING.md]: Is `php7.0-fpm` required if you just run cli? I thought that was only required with nginx.

Comment (lewisgoddard) [inline .github/CONTRIBUTING.md]: I'm not sure, possibly not. I can't really try it.

**Comment by btkostner** [comment]
Looks good to me, if you want to just double check the small edits I made.

**Comment by lewisgoddard** [comment]
That all looks fine to me, although people don't technically need composer installed unless we're going to remove the libraries from `backend` in the future (which wouldn't be the worst idea).

**Comment by btkostner** [comment]
The npm `composer` package was a completely different project. 

I thought that was the plan with composer. Remove all of the libraries so we only have the code _we_ have written in the repository.

**Comment by lewisgoddard** [comment]
Long term, sure, but we can leave that for a separate time. Lets keep this loki-targeted requests short, or it will just be more breakage.

**Pull request merged**